src/app/screens/Receive.tsx

```diff
@@ -75,6 +101,21 @@ function Receive() {
             label={copyLabel}
           />
         </div>
+        <div className="flex justify-center">
+          {pollingForPayment && (
+            <div className="flex items-center space-x-2">
+              <Loading />
+              <span>waiting for payment...</span>
+            </div>
+          )}
+          {!paid && !pollingForPayment && (
+            <Button
+              onClick={() => checkPayment(invoice.rHash)}
+              label="Check payment status"
+            />
+          )}
+          {paid && <p>Payment received!</p>}
```

Comment:
I think we should then also hide the QR code or maybe have some kind of ✅ overlay. 
or maybe @johnsBeharry confetti?

Reply:
is this too much of a dependency?

Reply:
Agree, yeah I like the "fun-factor" of that and think it's a small lib to add :)

Reply:
yeah, we need more fun factors! :)

Reply:
> Agree, yeah I like the "fun-factor" of that and think it's a small lib to add :)

I also agree, confetti are nice, at least in the beginning. Once I use the app five times a day the confetti animation might get a bit old, but for now I think it's great!